src/components/v2/appDetails/k8Resource/nodeType/Node.component.tsx

```diff
@@ -311,12 +320,12 @@ function NodeComponent({
 
     return (
         <>
-            {selectedNodes && selectedNodes.length > 0 && (
```

Comment:
is this intentional?

Reply:
Yes, if we don't remove this check these old/new pods tabs will disappear when node length is zero

Reply:
As per our discussion,  we are not making changes again as this is done intensionally